app/src/main/java/com/github/swent/echo/viewmodels/authentication/CreateProfileViewModel.kt

```diff
@@ -94,6 +100,17 @@ constructor(
                     _tagList.value = userProfile.tags
                     _committeeMember.value = userProfile.committeeMember
                     _associationSubscriptions.value = userProfile.associationsSubscriptions
+                    val pictureByteArray = repository.getUserProfilePicture(userId)
+                    _picture.value =
+                        if (pictureByteArray != null) {
+                            BitmapFactory.decodeByteArray(
+                                pictureByteArray,
+                                0,
+                                pictureByteArray.size
+                            )
+                        } else {
+                            null
```

Comment:
Similarly, the else block is not required here because the default value is null already.